datumaro/plugins/coco_format/extractor.py

```diff
@@ -1,13 +1,13 @@
-# Copyright (C) 2019-2021 Intel Corporation
+# Copyright (C) 2019-2022 Intel Corporation
 #
 # SPDX-License-Identifier: MIT
 
-from collections import OrderedDict
+from typing import Any
 import json
 import logging as log
 import os.path as osp
 
-from pycocotools.coco import COCO
+from attrs import define
```

Comment:
Using the `attrs` package requires bumping the lower bound for the `attrs` dependency to 21.3.

Reply:
Done